ccswbs/gus
# Add Accordion and Tabs widgets to Section

# Summary of changes
The basic page template for majors is supposed to match https://www.uoguelph.ca/dev/programs/environmental-management-major - this requires that the Tabs widget be made available to Sections. The Accordion widget is also being added for use in future layouts.

## Frontend
- Accordion widget added to `sectionWidgets.js`
- Tabs widget added to  `sectionWidgets.js`
- `field_section_column` added to GraphQL queries in `accordion.js` and `pageTabs.js`
- Schema updated in `gatsby-node.js`
- Packages upgrade to latest versions (incidental change)

## Backend
- Accordion and Tabs paragraph types made available to Section Widget
- Section Column field added to Accordion and Tabs paragraph types

# Test Plan

Drupal multidev: https://majors-chug.pantheonsite.io/
 
- Go to https://tqtest.gatsbyjs.io/majors-template
- Verify the overall layout is consistent with https://www.uoguelph.ca/dev/programs/environmental-management-major
- Go to https://tqtest.gatsbyjs.io/test-widgets
- Verify the Accordion and Tabs widgets work as expected

## Known issues
- The `Apply Now` button on https://tqtest.gatsbyjs.io/majors-template uses the button widget and, due to hard-coded classes, is narrower than the same button on https://www.uoguelph.ca/dev/programs/environmental-management-major - altering this behavior is beyond the scope of this pull request, but a workaround could involve using a general text widget in full html format instead (which in turn would permit adding a button with different classes).
